frontend/lib/src/components/core/Block/utils.test.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ElementNode } from "@streamlit/lib/src/AppNode"
+import { ScriptRunState } from "@streamlit/lib/src/ScriptRunState"
+
+import { isElementStale } from "./utils"
+
+describe("isElementStale", () => {
+  // @ts-expect-error
+  const node = new ElementNode(null, null, "myScriptRunId", "myFragmentId")
+
+  it("Returns true if scriptRunState is RERUN_REQUESTED", () => {
```

Comment:
super-nit: it looks like its more common to start the `it` string with a lower-case letter. If so, for general code hygiene we should keep it uniformed